src/library.js

```diff
@@ -1488,8 +1488,15 @@ LibraryManager.library = {
 #endif
 
   __cxa_pure_virtual: function() {
+#if !MINIMAL_RUNTIME
     ABORT = true;
+#endif
```

Comment:
if this pattern is common, we could replace these three lines with `{{{ makeAbort() }}}` and that function in parseTools.js would `return MINIMAL_RUNTIME ? "" : ABORT = true;`

Reply:
Searching through the codebase, there is only one other location like that - I'll gate that away as well.

In general I think it is better to have complexity in src/library_x.js side rather than the generic parseTools.js/jsifier.js tooling side - which  would be good to be general and simple and agnostic to usage patterns. That way when the src/library_x.js files mutate under refactoring, complexity in the JS linker itself remains low.